rapid7_insightvm/plugin.spec.yaml

```diff
@@ -3723,7 +3820,28 @@ actions:
         type: boolean
         required: true
         example: True
-
+  asset_vulnerability_solution:
+    title: Get Asset Vulnerability Solutions
+    description: Returns the highest-superceding rollup solutions for a list of vulnerabilities on an asset
+    input:
+      asset_id:
+        title: Asset ID
+        description: The identifier of the asset
+        type: integer
+        required: true
+        example: 423
+      vulnerability_ids:
+        title: Vulnerability IDs
+        description: A list of identifiers of the vulnerabilities
+        type: "[]string"
+        required: true
+        example: ["flash_player-cve-2017-11305"]
+    output:
+      vulnerabilities_solution:
+        title: Vulnerabilities Solutions
+        description: Highest-superceding rollup solutions for a vulnerabilities on an asset
```

Comment:
The title and description are referring to two different quantities of solutions/vulnerabilities - the description has "solutions for a vulnerabilities on an asset" while the title makes it seem like it is multiple solutions for multiple vulnerabilities. I'm not sure how this is supposed to read, but this needs to be clarified between the title and description.